### PR TITLE
Add pull request (PR) template

### PR DESCRIPTION
**Pull request by josh-wong**
## Description

This PR updates the PR template in this repository. The template is for helping to provide context for people who are involved with PRs and for future reference.

## Related issues and/or PRs

N/A

## Changes made

Created the pull request template in the **.github** folder.

## How these changes were tested

Checked that this PR template looks as expected in GitHub.

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

N/A

### CODE REVIEW

**Review by supl — APPROVED** [review]
Thank you, Josh 🙇

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+> Provide any additional information or notes that may be relevant to the reviewers or stakeholders.
+
+## Release notes
+
+> Provide release note text for this PR based on the guidelines at [Guidelines for Writing Release Notes](https://developers.scalar-labs.com/docs/style-guide/release-notes/).
+> 
+> For example:
+> 
+> Added validation for the Consensus Commit mutation operation.
```
Comment (kota2and3kan) [inline .github/pull_request_template.md:38]:
```suggestion
```

We don't have any release notes for `scalar-admin`.
So, I think we can remove this section at this time.
However, if we have any plans to create release notes in the future, I think it's OK to remain this section.

@supl 
What do you think?

**Review by kota2and3kan — COMMENTED** [review]
Overall looks good to me, but I have one suggestion.

@supl
Sorry, I want to discuss release note with you.
Could you please take a look again?

File: .github/pull_request_template.md (hunk 1 above)
Comment (supl) [reply .github/pull_request_template.md:38]: @kota2and3kan 
@josh-wong 

Thank you for pointing out this concern.

scalar-admin currently is a tool and library that is used by Scalar projects.
- scalar
- scalardb
- scalar-manager
- scalar-admin-k8s

It is not directly used by users so we don't have release notes (this saves maintenance costs).

In this assumption, I agree with Yokoyama-san. If we would just leave `N/A` to the `Release notes` section most of the time, then maybe it's easier to just remove it.

We can add this section back anytime whenever we need to provide release notes for some reason.

Comment (josh-wong) [reply .github/pull_request_template.md:38]: Got it! That makes sense. I've removed this section.

**Review by supl — APPROVED** [review]
LGTM.
Thank you for the revision.

File: .github/pull_request_template.md (hunk 1 above)
Comment (kota2and3kan) [reply .github/pull_request_template.md]: @supl 
Thank you for your answer!

> If we would just leave N/A to the Release notes section most of the time, then maybe it's easier to just remove it.
> We can add this section back anytime whenever we need to provide release notes for some reason.

I agree. Thank you!

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**